internal/clients/grpc/auth/client.go

```diff
@@ -1,28 +1,28 @@
 package auth
 
 import (
-	"github.com/mainflux/mainflux"
-	authapi "github.com/mainflux/mainflux/auth/api/grpc"
 	grpcClient "github.com/mainflux/mainflux/internal/clients/grpc"
 	"github.com/mainflux/mainflux/internal/env"
 	"github.com/mainflux/mainflux/pkg/errors"
+	"github.com/mainflux/mainflux/users/policies"
+	authapi "github.com/mainflux/mainflux/users/policies/api/grpc"
 )
 
 const envAuthGrpcPrefix = "MF_AUTH_GRPC_"
 
```

Comment:
This can be renamed to `MF_USERS_GRPC_` , Since `MF_USERS_GRPC_PORT` is used in `users` service for gRPC port 

The package can be renamed to `users`